### ISSUE TITLE
Test coverage integration

### ISSUE THREAD

**Issue by sfnelson** [cursive-ide/cursive#167]
With all the fantastic testing support cursive is getting (soon?) it would be great if it could also get test coverage support.

There is at least one clojure test coverage tool available already (https://github.com/lshift/cloverage) or it might be more straightforward to use an existing tool that already has IntelliJ integration.

**Comment by boschi** [comment]
Having this integration with Cloverage and IntelliJ would be quite nice!

**Comment by aboy021** [comment]
Is this something we might still see in the future?

**Comment by cursive-ide** [comment]
@aboy021 Yes it is, I actually had a go at integrating it with the IntelliJ coverage support recently. Unfortunately it was much harder than I expected, due to the IntelliJ JVM coverage engine being very class-focused. So I parked it but I do want to come back to it.
